Gruntfile.js

```diff
@@ -232,10 +237,102 @@ module.exports = function( grunt ) {
 					from: /^(\*\*|)Stable tag:(\*\*|)(\s*?)[a-zA-Z0-9.-]+(\s*?)$/mi,
 					to: '$1Stable tag:$2$3<%= pkg.version %>$4'
 				} ]
+			},
+			version_docs: {
+				src: [
+					'.dev/docs/themes/godaddy/*.html',
+					'.dev/docs/apigen/theme-godaddy/*.latte'
+				],
+				overwrite: true,
+				replacements: [ {
+					from: /Primer Theme v[a-zA-Z0-9\.\-\+]+/m,
+					to: 'Primer Theme v' + pkg.version
+				} ]
+			},
+			intro_docs: {
+				src: [
+					'.dev/docs/en/intro.md',
+				],
+				overwrite: true,
+				replacements: [ {
+					from: '[![Build Status](https://travis-ci.org/godaddy/wp-primer-theme.svg?branch=master)](https://travis-ci.org/godaddy/wp-primer-theme) [![Built with Grunt](https://cdn.gruntjs.com/builtwith.svg)](http://gruntjs.com)',
+					to: ''
+				},
+				{
+					from: '  ',
+					to: ' <br />'
+				},
+				{
+					from: '<!-- DO NOT EDIT THIS FILE; it is auto-generated from readme.txt -->',
+					to: ''
+				} ]
+			}
+		},
+
+		sass: {
+			options: {
+				precision: 5,
+				sourceMap: false
+			},
+			dist: {
+				files: [
+					{
+						'style.css': '.dev/sass/style.scss',
+						'editor-style.css': '.dev/sass/editor-style.scss'
+					},
+					{
+						expand: true,
+						cwd: '.dev/sass/admin',
+						src: [ '*.scss' ],
+						dest: 'assets/css/admin',
+						ext: '.css'
+					}
+				]
+			}
+		},
+
+		output: {
+			doc_help: {
+				content : {
+					mode : 'log',
+					before: 'Before you can compile the documentation there are a few dependencies that need to be installed. \r\n\r\n' +
+						'First, ensure your system has Python installed. You can check if Python is installed by running the following command \r\n\r\n' +
+						'$ python --version \r\n\r\n' +
+						'Once Python is installed you need to install pip. You can install pip by running the following command \r\n\r\n' +
+						'$ sudo easy_install pip \r\n\r\n' +
+						'Finally you need to install the sphinx dependencies. You can do this by running the following command \r\n\r\n' +
+						'$ sudo pip install -r .dev/docs/requirements.txt \r\n\r\n' +
+						'After all of the dependencies are installed you can run "grunt docs" to compile the documentation into the .dev/docs/en/documentation/ directory.',
+					func : 'writeln',
+					color: 'white',
+					styles: []
+				},
 			}
+		},
+
+		shell: {
+			sphinx: [
+				'cd .dev/docs',
```

Comment:
@EvanHerman what if we just add `easy_install pip` and `pip install -r requirements.txt` to the top of this array? macOS comes with Python 2.7, so checking for it really isn't necessary.